client/src/com/aerospike/client/IAerospikeClient.java

```diff
@@ -33,6 +33,9 @@
 import com.aerospike.client.listener.ExistsArrayListener;
 import com.aerospike.client.listener.ExistsListener;
 import com.aerospike.client.listener.ExistsSequenceListener;
+import com.aerospike.client.listener.IndexExistsListener;
+import com.aerospike.client.listener.IndexListener;
```

Comment:
Probably you could revert code formatting changes?

Reply:
fixed